spec/models/hmpps_api/offender_spec.rb

```diff
@@ -235,14 +235,14 @@
         end
       end
 
-      describe '#ldu' do
-        let(:field) { :ldu }
+      describe '#ldu_name' do
+        let(:field) { :ldu_name }
 
-        it { is_expected.to be(case_info.team.local_divisional_unit) }
+        it { is_expected.to be(case_info.team.local_divisional_unit.name) }
       end
 
       describe '#team' do
```

Comment:
This describe block should change to reflect the new method name:
```suggestion
      describe '#team_name' do
```